.github/workflows/check-release.yml

```diff
@@ -0,0 +1,29 @@
+name: Check Release
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    branches: ["*"]
+
+jobs:
+  check_release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Base Setup
+        uses: jupyterlab/maintainer-tools/.github/actions/base-setup@v1
+      - name: Install Dependencies
+        run: |
+          pip install -e .
```

Comment:
Normally it would be better to remove this step as it is not needed anymore.

Reply:
Having a `pip install -e .` only in the check release workflow but not in the "Publish Release" workflow (the one used to do the real release) can add unexpected side effects (some dev dependencies missing in the actual release workflow.

Reply:
Thanks @jtpio